kfrederick_dev: 116_Jorgensen_YKDE review

kristenfrederick opened the pull request:
116_Jorgensen_YKDE is ready for review. #57

kristenfrederick committed:
Merge branch 'master' of https://github.com/ISCN/SOCDRaHR2 into kfrederick_dev

kristenfrederick committed:
Merge branch 'master' of https://github.com/ISCN/SOCDRaHR2 into kfrederick_dev

kristenfrederick committed:
Adding Jorgensen_YKDE to the bibliography and references, fixing Alaska map, commenting changes #57

kristenfrederick committed:
Merge branch 'master' of https://github.com/ISCN/SOCDRaHR2 into kfrederick_dev

kristenfrederick committed:
Fixed Jorgensen_YKDE citation and finished comments about ISCN3.5 changes #57

ktoddbrown commented on data_reports/116_Jorgensen_YKDE.Rmd:
````diff
@@ -1,30 +1,205 @@
 
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
+The Jorgensen YKDE data set in ISCN3 contains `r nrow(dataset_layer)` layer-level rows and `r nrow(dataset_profile)` profile-level information rows after cleaning for ISCN3.5.
````
Need to actually put the number here since the dataset_layer and dataset_profile are not ingested yet.

ktoddbrown commented on data_reports/116_Jorgensen_YKDE.Rmd:
```diff
@@ -1,30 +1,205 @@
-knitr::opts_chunk$set(echo = TRUE)
+The Jorgensen YKDE data set in ISCN3 contains `r nrow(dataset_layer)` layer-level rows and `r nrow(dataset_profile)` profile-level information rows after cleaning for ISCN3.5.
+Rows weren't removed from ISCN3 because there were no SOC computations that needed removing.
+Columns were removed from ISCN3 because ISCN3 filled in the soil carbon stocks where bulk density and organic carbon was provided. 
```
This contradicts with the line above.

ktoddbrown commented on index.Rmd:
```diff
@@ -51,19 +51,21 @@ rm(ISCN3) #memory management, ISCN3 is big
 #csv data should be archived by summer 2021
 
-citation_raw <- read_delim(file.path(data_dir, 'ISCN3_citation.csv'), delim = ';', col_types = strrep('c', times = 12)) %>% 
+citation_raw <- read_delim('~/ISCN/ISCN3_citation.csv', delim = ';', col_types = strrep('c', times = 12)) %>% 
```
Use file.path instead here.

ktoddbrown commented on index.Rmd:
````diff
@@ -51,19 +51,21 @@ rm(ISCN3) #memory management, ISCN3 is big
+layer_raw <- vroom::vroom('~/ISCN/ISCN3_layer.csv', col_types = strrep('c', times = 95))
 ```
 
+=======
````
Remove this like, likely from a merge

kristenfrederick committed:
Switching back to file path on Index

Fixing the code that calls the data to use file.path

kristenfrederick committed:
Update 116_Jorgensen_YKDE.Rmd

Fixing the row and column count for layer data and fixing the explanation of the removed columns

kristenfrederick committed:
Update index.Rmd

Removing the line that occurred during a merge

kristenfrederick commented:
The pull request is ready for a second review

ktoddbrown reviewed (approved):
Looks good!
